jetty-core/jetty-io/src/main/java/org/eclipse/jetty/io/Content.java

```diff
@@ -571,6 +555,23 @@ static Chunk slice(Chunk chunk)
          */
         boolean isLast();
 
+        /**
+         * <p>Returns a new {@code Chunk} whose {@code ByteBuffer} is a slice of the
+         * {@code ByteBuffer} of the source {@code Chunk}.</p>
+         * <p>The returned {@code Chunk} retains the source {@code Chunk} and it is linked
+         * to it via {@link #from(ByteBuffer, boolean, Retainable)}.</p>
```

Comment:
Update javadoc with "unless is EOF or EMPTY" or similar.

Reply:
Done.